Documentation/Storage-Configuration/Block-Storage-RBD/block-storage.md

```diff
@@ -59,7 +59,13 @@ parameters:
     # RBD image format. Defaults to "2".
     imageFormat: "2"
 
-    # RBD image features. Available for imageFormat: "2". CSI RBD currently supports only `layering` feature.
+    # RBD image features
+    # Available for imageFormat: "2". Older releases of CSI RBD
+    # support only the `layering` feature.  The Linux kernel (KRBD) supports the
+    # full complement of features as of 5.4
+    # `layering` alone corresponds to Ceph's bitfield value of "2" ;
+    # `layering` + `fast-diff` + `image-map` + `deep-flatten` + `exclusive-lock` together
```

Comment:
Please mention passing this as comma-separated values in imageFeatures parameter.

Reply:
I'm pushing elaboration. PTAL